src/doc/trpl/structs.md

```diff
@@ -47,3 +54,36 @@ fn main() {
 ```
 
 This will print `The point is at (5, 0)`.
+
+Rust does not support mutability at the field level, so you cannot write
```

Comment:
Maybe something like "at the language level" could be thrown in here? Technically `Cell` and `RefCell` are tasked with the purpose of field-level mutability.